packages/web3-errors/src/errors/rpc_errors.ts

```diff
@@ -33,14 +33,17 @@ import {
 	ERR_RPC_UNAVAILABLE_RESOURCE,
 	ERR_RPC_UNSUPPORTED_METHOD,
 } from '../error_codes.js';
+import { RpcErrorMessages } from './rpc_error_messages.js';
+
+const genericRpcMessage = 'An Rpc error has occured with a code of *code*';
```

Comment:
I think it would make more sense if you moved `genericRpcMessage to `./rpc_error_messages.ts`

Reply:
Done.